source/sync/app-builder/tiered-device-sync/index.txt

```diff
@@ -0,0 +1,379 @@
+:noprevnext:
+:orphan:
+
+.. meta::
+   :robots: noindex, nosnippet
+
+.. _tiered-device-sync-guide:
+
+========================
+Tiered Device Sync Guide
+========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+This guide details how to set up a Tiered Device Sync application while 
+this feature is in pre-alpha. This guide is intentionally not public,
+and is intended for internal MongoDB use.
```

Comment:
Because there are multiple procedures, consider adding an "Über Procedure" (that's what a previous team I worked on called them 😄). Essentially, list each of the procedures and link to them.

This helps make the relationship between the procedures clearer at a glance. But it may not be needed for this internal audience.

Reply:
I started to do this, but then realized that isn't a paradigm we really follow anywhere else in our docs and it struck me as awkward and weirdly repetitive. Is there a public example you can link me to of where you think this is done well, @krollins-mdb ?

Reply:
Hmm. I swear Algolia had some examples of this, but I couldn't find any when I briefly looked. Unfortunately, the docs at the company we did this are all locked behind auth for customers.

I briefly looked through MMoS and the Google Developer doc style guide and couldn't find anything. Sorry I can't find any examples for you. I'm going to post in Write the Docs. 😆 

I think there's value in it to show that procedures need to be done in a specific order. But it seems like lots of docs don't make that super clear.